src/networking/responses/branding-response.ts

```diff
@@ -7,5 +7,4 @@ export type BrandingInfoResponse = {
   id: string;
   app_name: string | null;
   support_email?: string | null;
-  gateway_tax_collection_enabled: boolean;
```

Comment:
Why removing this?
Isn't it already returned by the backend?

Reply:
This is bizarre but I was getting type check errors in my runtime. I will review again.

Reply:
Reverted just in case, unsure why my env was complaining at the time.